tests/conftest.py

```diff
@@ -309,3 +321,29 @@ def __init__(
         endpoints_path: Text = "",
     ) -> None:
         super().__init__(tracker_store, event_broker, endpoints_path)
+
+
+def mock_async(
+    monkeypatch: MonkeyPatch, target: Any, name: Text, return_value: Any = None
```

Comment:
Just a thought, but this could be a fixture and `monkeypatch` would be injected automatically. Don't feel strongly though.

Reply:
mhm, but that then we couldn't set the `return_value` on the fly 😬

Reply:
I think the pattern would be to have the fixture return a function that already has `monkeypatch` in its closure.

Reply:
Mhm, or something like that: https://stackoverflow.com/a/32498408/3429596 . Seems even simpler

Reply:
used the things from above